journals/apps/journals/admin.py

```diff
@@ -23,3 +35,21 @@ class JournalAccessAdmin(admin.ModelAdmin):
 
 # Default admin pages below
 admin.site.register(Organization)
+
+
+class VideoModelAdmin(ModelAdmin):
+    """
+    Video model admin
+    """
+    model = Video
+    menu_label = 'Videos'
+    menu_icon = 'media'
+    menu_order = 400
+    add_to_settings_menu = False
+    exclude_from_explorer = False
+    permission_helper_class = VideoPermissionHelper
+    index_view_class = VideoIndexView
+    form_fields_exclude = ('collection',)
```

Comment:
can we make Block Id and Source Course Run fields read-only in the admin edit view? Don't want these to be mucked with.

Reply:
@bill-filler add `source_course_run` and `block_id` to excluded fields list.